aten/src/ATen/native/native_functions.yaml

```diff
@@ -308,6 +308,10 @@
 
 - func: select(Tensor self, int64_t dim, int64_t index) -> Tensor
 
+- func: relu(Tensor self) -> Tensor
```

Comment:
Should this also be function-only?

Reply:
Yeah it will generate both. I just mentioned because I saw rrelu and selu are function-only. Maybe @colesbury has more to say on this.